openapi.yaml

```diff
@@ -25,10 +25,10 @@ paths:
         This is the primary interface to lookup the content required to render
         a page on GOV.UK
 
-        It accepts an input of the path of the page you are looking up. Eg if
-        you were looking up travel advice for Thailand which is hosted on GOV.UK
-        at `https://www.gov.uk/foreign-travel-advice/thailand` you would make
-        a request to
+        It accepts an input of the path of the page you are looking up. For
+        example if you were looking up travel advice for Thailand which is
```

Comment:
should we have a comma after example here?

Reply:
Yep, fixed!